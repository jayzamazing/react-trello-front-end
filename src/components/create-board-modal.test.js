'use strict';
import React from 'react';
import toJson from 'enzyme-to-json';
import {shallow} from 'enzyme';
import CreateBoardModal from './create-board-modal';
//testing for the create-board-modal
describe('create-board-modal', () => {
  it('should render', () => {
    shallow(<CreateBoardModal isOpen={true}/>);
  });
  //take snapshot
  it('snapshot should match', () => {
    const wrapper = shallow(<CreateBoardModal isOpen={true}/>);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
