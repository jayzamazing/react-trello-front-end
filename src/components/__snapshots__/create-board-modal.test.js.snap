// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create-board-modal snapshot should match 1`] = `
<Modal
  ariaHideApp={true}
  bodyOpenClassName="ReactModal__Body--open"
  className={
    Object {
      "afterOpen": "create-board-after-open",
      "base": "create-board",
      "beforeClose": "create-board-before-close",
    }
  }
  closeTimeoutMS={0}
  contentLabel="Create Board Modal"
  isOpen={true}
  overlayClassName={
    Object {
      "afterOpen": "create-board-overlay-after-open",
      "base": "create-board-overlay",
      "beforeClose": "create-board-overlay-before-close",
    }
  }
  parentSelector={[Function]}
  portalClassName="ReactModalPortal"
  shouldCloseOnEsc={true}
  shouldCloseOnOverlayClick={true}
  shouldFocusAfterRender={true}
  shouldReturnFocusAfterClose={true}
>
  <ReduxForm />
</Modal>
`;
